src/com/company/reverse/ReverseTest.java

```diff
@@ -0,0 +1,12 @@
+package com.company.reverse;
+
+public class ReverseTest {
```

Comment:
Please rename this class to `Application`

Reply:
Done